src/snyk/cli/mcp/mcp.ts

```diff
@@ -115,13 +121,19 @@ export async function configureWindsurf(vscodeContext: vscode.ExtensionContext,
     Logger.error('Failed to update Windsurf MCP config');
   }
 
+  const globalPath = path.join(os.homedir(), '.codeium', 'windsurf', 'memories', 'global_rules.md');
+  const localPath = path.join('.windsurf', 'rules', 'snyk_rules.md');
   try {
-    if (!securityAtInception.publishSecurityAtInceptionRules) return;
+    if (!securityAtInception.publishSecurityAtInceptionRules) {
+      // Delete rules from both project and global
+      await deleteLocalRulesForIde(localPath);
+      await deleteGlobalRules(globalPath);
+      return;
+    }
```

Comment:
can we extract these bits? looks like a bit of duplication. I think only the filePathes differ, the rest is the same, right?